src/PhpParser/NodeTransformer.php

```diff
@@ -136,7 +136,7 @@ private function splitMessageAndArgs(FuncCall $sprintfFuncCall): ?SprintfStringA
      */
     private function splitBySpace(string $value): array
     {
-        $value = str_getcsv($value, ' ');
+        $value = str_getcsv($value, ' ', escape: '\\');
```

Comment:
str_getcsv(): the $escape parameter must be provided as its default value will change

Reply:
the escape parameter default is already `\\`, so this unnecessary. see documentation https://www.php.net/manual/en/function.str-getcsv.php

Reply:
![image](https://github.com/user-attachments/assets/18aeed4c-73af-4b40-9412-0f5c54b3040f)
In PHP8.4 if escape arg wasn't provided the error is generated
> str_getcsv(): the $escape parameter must be provided as its default value will change

Reply:
again, test is needed to avoid blind patch